docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on {VscodePluginName}
+
+You can install the {ProductShortName} extension on {VscodePluginName} on Linux, Windows or iOS.
+
+.Prerequisites
+
+* link:https://code.visualstudio.com[VS Code installed]
+* 4 GB of RAM (8 GB recommended)
+* JDK 11+
+* `JAVA_HOME` set to `jdk11`
+
+.Procedure
```

Comment:
WHat about MTA-CLI? When you install, the extension, the UI searches for MTA-CLI and has a button to install it. These steps have to be part of the MTA CLI. MTA CLI is NOT a usage preprequisite. It is part of the installation because it is installed through the UI before the user exits the extension. Try it and you will see what I mean.